include/bluetooth/iso.h

```diff
@@ -0,0 +1,256 @@
+/** @file
+ *  @brief Bluetooth ISO handling
+ */
+
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_ISO_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_ISO_H_
+
+/**
+ * @brief ISO
+ * @defgroup bt_iso ISO
+ * @ingroup bluetooth
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <sys/atomic.h>
+#include <bluetooth/buf.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/hci.h>
+
+/** @def BT_ISO_CHAN_SEND_RESERVE
+ *  @brief Headroom needed for outgoing buffers
+ */
+#define BT_ISO_CHAN_SEND_RESERVE (CONFIG_BT_HCI_RESERVE + \
+				  BT_HCI_ISO_HDR_SIZE + \
+				  BT_HCI_ISO_DATA_HDR_SIZE)
+
+struct bt_iso_chan;
+
+/** @brief Life-span states of ISO channel. Used only by internal APIs
+ *  dealing with setting channel to proper state depending on operational
+ *  context.
+ */
+enum {
+	/** Channel disconnected */
+	BT_ISO_DISCONNECTED,
+	/** Channel bound to a connection */
+	BT_ISO_BOUND,
+	/** Channel in connecting state */
+	BT_ISO_CONNECT,
+	/** Channel ready for upper layer traffic on it */
+	BT_ISO_CONNECTED,
+	/** Channel in disconnecting state */
+	BT_ISO_DISCONNECT,
+};
+
+/** @brief ISO Channel structure. */
+struct bt_iso_chan {
+	/** Channel connection reference */
+	struct bt_conn			*conn;
+	/** Channel operations reference */
+	struct bt_iso_chan_ops		*ops;
+	/** Channel QoS reference */
+	struct bt_iso_chan_qos		*qos;
+	/** Channel data path reference*/
```

Comment:
comment style

Reply:
Can you be more specific? CI didn't catch any documentation style problem in these.